extensions/amp-app-banner/0.1/test/test-amp-app-banner.js

```diff
@@ -270,8 +276,17 @@ describe('amp-app-banner', () => {
       });
     });
 
-    it('should remove banner if safari', () => {
+    it('should remove banner if safari and not embedded', () => {
```

Comment:
Yes unfortunately. We remove Safari if the document is not embedded (because Safari's builtin mechanism will kick in and display their own app banner). And the second one (new change) is due to a bug in Safari which is preventing us from going to the app-store.

I know this is equivalent to hiding it on Safari all together but I wanted to keep the two conditions separate to display different warning or info messages to explain why the banner was not shown - and because we'll be working on a better fix and this case will be dropped in future.